conf.py

```diff
@@ -186,10 +187,12 @@
     # The ``CodeToRestSphinx`` extension creates a file named
     # ``sphinx-enki-info.txt``, which should be ignored by Sphinx.
     "sphinx-enki-info.txt",
+    # TODO: Notes here. I assume this is produced by the coverage run in the test suite? A link to that point in the code would be nice.
     "test/htmlcov",
-    "alembic/versions",
-    "**/.mypy_cache",
+    # TODO: Why ignore this?
     "bookserver/templates",
+    # TODO: notes here on how Alembic works. I think it stores migration data here.
```

Comment:
I think this is a weird place to document alembic.  We should have an rst file in the alembic folder that describes it and how we use it.

Reply:
I've updated the docs.